docs/en/guides/pipeline-integrations/development/sgtk-developer-app.md

```diff
@@ -0,0 +1,257 @@
+---
+layout: default
+title: Developing apps
+pagename: sgtk-developer-app
+lang: en
+---
+
+# Developing your own app
+
+## Introduction
+
+This guide outlines what a Toolkit app is, and covers how to create your own, and also explains some of the basics around app development.
+Shotgun Pipeline Toolkit is not only a collection of apps and engines maintained by Shotgun Software, it is also a development platform that makes it easy to quickly build custom pipeline tools!
+
+- [What is a Toolkit app?](#what-is-a-toolkit-app)
+- [Creating your own app](#creating-your-own-app)
+
+Steps:
+1. [Creating a development sandbox](#part-1---creating-a-development-sandbox)
+2. [Forking or downloading the starter app repository](#part-2---forking-or-downloading-the-starter-app-repository)
+3. [Adding the app to your config](#part-3---adding-the-app-to-your-config)
+4. [Developing the app](#part-4---developing-the-app)
+    - [Anatomy of the Template Starter App](#anatomy-of-the-template-starter-app)
+    - [Configuration settings](#configuration-settings)
+    - [Frameworks](#frameworks)
+    - [Reloading your changes](#reloading-your-changes)
+5. [Testing](#part-5---testing)
+6. [Preparing your first release](#part-6---preparing-your-first-release)
+
+Additional info:
+- [Modifying an existing app](#modifying-an-existing-app)
+    - [Pull requests](#pull-requests)
+
+## What is a Toolkit app?
+
+A Toolkit app can be defined as follows:
+
+- A Tool that is usually run by an end-user, using the Shotgun integrations.
+- They usually have a graphical user interface with which the user can guide the app's operations, but it doesn't have to, it can also be a command registered with the integrations, triggered by the user from the Shotgun menu in the software.
+- They can have an API/public methods in which other processes or apps could interact.
+- They can be multi-platform, and software agnostic.
+- It can be set up to be configured differently per [environment](https://developer.shotgunsoftware.com/487a9f2c/?title=Environment+Configuration+Reference#what-is-an-environment).
```

Comment:
I'd change the "it" to "they" here – now that we've moved to plural, let's stick with it.